swift/convert-phone-number-to-country-name/Sources/index.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  File.swift
+//  
+//
+//  Created by Matej Bačo on 04/03/2022.
+//
+
+import Foundation
+import Appwrite
+import AppwriteModels
+
+enum ExampleError : Error {
+  case missing(String)
+  case invalid(String)
+}
+
+/*
+Globaly-scoped cache used by function. Example value:
```

Comment:
```suggestion
Globally scoped cache used by function. Example value:
```
Please check the other readmes as well